index.js

```diff
@@ -35,7 +35,7 @@ function resolveUrlLoader(content, sourceMap) {
   //  however we need to match to the sass-loader and it does not do so
   var loader     = this,
       filePath   = loader.context,
-      outputPath = path.resolve(loader.options.output.path);
+      outputPath = path.resolve(loader.options.context);
```

Comment:
Where does the `contextPath` come from? There is only a `context` property in the webpack config file.
https://webpack.github.io/docs/configuration.html#context